src/core/ext/xds/xds_client.cc

```diff
@@ -495,8 +495,11 @@ class XdsClient::ChannelState::StateWatcher
                 parent_->xds_client(), parent_->server_.server_uri.c_str(),
                 status.ToString().c_str());
         parent_->xds_client_->NotifyOnErrorLocked(
-            GRPC_ERROR_CREATE_FROM_STATIC_STRING(
-                "xds channel in TRANSIENT_FAILURE"));
+            GRPC_ERROR_CREATE_FROM_COPIED_STRING(
```

Comment:
Please use `GRPC_ERROR_CREATE_FROM_CPP_STRING()` here, to avoid an unnecessary copy.

Reply:
done